example-schema/relationships/HasYoungerSibling.yaml

```diff
@@ -1,4 +1,5 @@
 name: HasYoungerSibling
+relationship: HAS_YOUNGER_SIBLING
```

Comment:
There should only be a sinlge object, which has both a from and a to. Maybe I need to explain in more detail